src/API/API.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const baseURL = 'https://api.spacexdata.com/v3/missions';
+
+const fetchMissions = axios.get(baseURL);
+
+export default fetchMissions;
```

Comment:
We can move all the API calls to this file later, to have all the asynchronous functionality there. We can delete the file for now and figure it out later as we are not using it right now.